solax/inverters/x3_hybrid_g4.py

```diff
@@ -27,7 +27,7 @@ class X3HybridG4(Inverter):
             vol.Required("data"): vol.Schema(
                 vol.All(
                     [vol.Coerce(float)],
-                    vol.Length(min=300, max=300),
+                    vol.Length(min=200, max=300),
```

Comment:
Not sure why this check was 300, 300? The response I get via LAN was 200 long so it works with this change.

I'm a bit confused though since my inverter worked directly via AP access on 5.8.8.8. Are the lengths different depending on how you connect?

Reply:
Let’s try it with your change and see how it affects the tests.

Thanks for raising this PR btw! Judging by the failed checks, I think you might need to run the formatter to make the checks go green.

Reply:
Right. I'll try (how do I run the formatter? 🙈 not a python dev..)

Seems like the file it complains about isn't touched by this PR fwiw

Reply:
I pushed a formatted version of that file